src/index.js

```diff
@@ -150,8 +155,101 @@ const renderLevel = (root, levelNumber) => {
         .join("")}
     </tr>
   `;
+
+  renderSettings = () => {  
+    const settingsHTML = `
+    <div id="modal">
+      <div class="modal-content">
+        <span class="close material-symbols-outlined">cancel</span>
+        <h1>Setting</h1>
+      </div>
+    </div>
+    `;
+    document.querySelector("#overlay-screen").innerHTML = settingsHTML;
+
+    document.querySelector(".close").addEventListener("click", () => {
+      document.querySelector("#modal").style.display = "none";
```

Comment:
Maybe setting `.innerHTML = ""` would be a better way to "close" the modal. The proposed approach looks OK at a glance, but it could easily add duplicate listeners if we're not careful to reset `.innerHTML`, so it simplifies things to just destroy the whole HTML content up front rather than hide it.